addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -267,6 +269,26 @@ def _compute_can_approve(self):
             else:
                 allocation.can_approve = True
 
+    def _inverse_date_from(self):
+        for record in self:
```

Comment:
Minor detail, for consistency within the file
```suggestion
        for allocation in self:
```
Valid for all 4 loops